lsqecc/pauli_rotations/circuit.py

```diff
@@ -89,11 +89,13 @@ def apply_transformation(self, start_index: int = 0) -> None:
         # Build a stack of pi/4 rotations
 
         for i in range(start_index, len(self)):
-            if isinstance(self.ops[i], PauliRotation) and self.ops[i].rotation_amount in {
-                Fraction(1, 4),
-                Fraction(-1, 4),
-            }:
-                quarter_rotation.append(i)
+            if isinstance(self.ops[i], PauliRotation):
+                temp = cast(PauliRotation, self.ops[i])
+                if temp.rotation_amount in {
```

Comment:
```suggestion
                if cast(PauliRotation, self.ops[i]).rotation_amount in {
```
Just fixing my code here 😅

Reply:
Yep :+1:  I remember us talking about `cast`